recipes-core/initscripts/initscripts-1.0/openxt-installer/urandom

```diff
@@ -0,0 +1,61 @@
+#! /bin/sh
```

Comment:
Saving the random seed doesn't make sense for an ephemeral installer image, so maybe remove this init script?

Reply:
SGTM